### PR TITLE
#1060 Renamed testcases in examples and make it point to examples. De…

### PR DESCRIPTION
**Pull request by gautierbureau**
…leted sources from distrib.

### CODE REVIEW

**Comment by gautierbureau** [comment]
This commit will need to be merged right before we do the next release. (I do it now to not forget it)

**Comment by gautierbureau** [comment]
@adriguir fyi

File: util/envDynawo.sh [hunk 1]
```diff
@@ -2026,18 +2014,31 @@ create_distrib() {
...
-  copy_sources
+  ln -s $DYNAWO_HOME/examples $DYNAWO_DEPLOY_DIR/examples
 
   if [ ! -x "$(command -v zip)" ]; then
     error_exit "You need to install zip command line utility."
   fi
 
+  if [ "`uname`" = "Linux" ]; then
+    if [ -x "$(command -v chrpath)" ]; then
+      if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" ]; then
```
Comment (rosiereflo) [inline util/envDynawo.sh:2025]: Needs to be removed

**Review by rosiereflo — CHANGES_REQUESTED** [review]
Given the time between the creation and the merge of this PR, could you please make sure this is still doing what is expected?

File: util/envDynawo.sh (hunk 1 above)
Comment (gautierbureau) [reply util/envDynawo.sh]: done

**Comment by gautierbureau** [comment]
> Given the time between the creation and the merge of this PR, could you please make sure this is still doing what is expected?

Yes it is ok !

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**